### ISSUE TITLE
The model result presented in the paper on SIMS cannot be replicated.  

### ISSUE THREAD

**Issue by BBQLXDM** [zehuiwu/MMML#8]
Hello, could you please tell me the model hyperparameters used on the SIMS dataset in your model? I am using an RTX 4090 GPU, which is samed in the paper. I used the default hyperparameter settings, but I cannot to reproduce the results on the SIMS dataset as shown in the paper.

**Comment by Pioneer-Weirdo** [comment]
I'm running the program with the following command on an RTX 4090 GPU:

`python run.py --batch_size 2 --lr 5e-6 --dataset mosi --tasks MTA --context True`
Here are the results I’m getting:

**TEST (highest val acc):**
Has0_acc_2: 0.8586
Has0_F1_score: 0.8576
Non0_acc_2: 0.8841
Non0_F1_score: 0.8838
Mult_acc_5: 0.5918
Mult_acc_7: 0.5160
MAE: 0.5941
Corr: 0.8642
Loss: 2.3815

**TEST (lowest val loss):**
Has0_acc_2: 0.8499
Has0_F1_score: 0.8472
Non0_acc_2: 0.8811
Non0_F1_score: 0.8797
Mult_acc_5: 0.5685
Mult_acc_7: 0.4840
MAE: 0.6027
Corr: 0.8602
Loss: 2.4808

😕Could you confirm if these results seem reasonable and if there are any adjustments I should make for optimal performance?
```TEST >> loss:  2.3815    M_loss:  0.5941   T_loss:  0.5993   A_loss:  1.188
M: >>  Has0_acc_2: 0.8586  Has0_F1_score: 0.8576  Non0_acc_2: 0.8841  Non0_F1_score: 0.8838  Mult_acc_5: 0.5918  Mult_acc_7: 0.5160  MAE: 0.5941  Corr: 0.8642 
T: >>  Has0_acc_2: 0.8673  Has0_F1_score: 0.8670  Non0_acc_2: 0.8872  Non0_F1_score: 0.8873  Mult_acc_5: 0.5612  Mult_acc_7: 0.4810  MAE: 0.5993  Corr: 0.8659 
A: >>  Has0_acc_2: 0.6910  Has0_F1_score: 0.6900  Non0_acc_2: 0.7027  Non0_F1_score: 0.7028  Mult_acc_5: 0.2930  Mult_acc_7: 0.2741  MAE: 1.1880  Corr: 0.4532 
TEST (highest val acc) : >>  Has0_acc_2: 0.8586  Has0_F1_score: 0.8576  Non0_acc_2: 0.8841  Non0_F1_score: 0.8838  Mult_acc_5: 0.5918  Mult_acc_7: 0.5160  MAE: 0.5941  Corr: 0.8642  Loss: 2.3815 
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 343/343 [00:54<00:00,  6.34it/s]
TEST >> loss:  2.4808    M_loss:  0.6027   T_loss:  0.586   A_loss:  1.2922
M: >>  Has0_acc_2: 0.8499  Has0_F1_score: 0.8472  Non0_acc_2: 0.8811  Non0_F1_score: 0.8797  Mult_acc_5: 0.5685  Mult_acc_7: 0.4840  MAE: 0.6027  Corr: 0.8602 
T: >>  Has0_acc_2: 0.8557  Has0_F1_score: 0.8541  Non0_acc_2: 0.8811  Non0_F1_score: 0.8803  Mult_acc_5: 0.5773  Mult_acc_7: 0.5000  MAE: 0.5860  Corr: 0.8677 
A: >>  Has0_acc_2: 0.6516  Has0_F1_score: 0.6500  Non0_acc_2: 0.6524  Non0_F1_score: 0.6521  Mult_acc_5: 0.2449  Mult_acc_7: 0.2376  MAE: 1.2922  Corr: 0.3512 
TEST (lowest val loss) : >>  Has0_acc_2: 0.8499  Has0_F1_score: 0.8472  Non0_acc_2: 0.8811  Non0_F1_score: 0.8797  Mult_acc_5: 0.5685  Mult_acc_7: 0.4840  MAE: 0.6027  Corr: 0.8602  Loss: 2.4808

**Comment by duanruixia** [comment]
> 我在 RTX 4090 GPU 上使用以下命令运行该程序：
> 
> `python run.py --batch_size 2 --lr 5e-6 --dataset mosi --tasks MTA --context True`以下是我得到的结果：
> 
> **测试（最高值累积）：**Has0_acc_2：0.8586 Has0_F1_score：0.8576 Non0_acc_2：0.8841 Non0_F1_score：0.8838 Mult_acc_5：0.5918 Mult_acc_7：0.5160 MAE：0.5941 Corr：0.8642 亏损：2.3815
> 
> **TEST（最低价值损失）：**Has0_acc_2：0.8499 Has0_F1_score：0.8472 Non0_acc_2：0.8811 Non0_F1_score：0.8797 Mult_acc_5：0.5685 Mult_acc_7：0.4840 MAE：0.6027 Corr：0.8602 损失：2.4808
> 
> 😕您能否确认这些结果是否合理，以及我是否应该进行任何调整以获得最佳性能？
> 
> ```
> M: >>  Has0_acc_2: 0.8586  Has0_F1_score: 0.8576  Non0_acc_2: 0.8841  Non0_F1_score: 0.8838  Mult_acc_5: 0.5918  Mult_acc_7: 0.5160  MAE: 0.5941  Corr: 0.8642 
> T: >>  Has0_acc_2: 0.8673  Has0_F1_score: 0.8670  Non0_acc_2: 0.8872  Non0_F1_score: 0.8873  Mult_acc_5: 0.5612  Mult_acc_7: 0.4810  MAE: 0.5993  Corr: 0.8659 
> A: >>  Has0_acc_2: 0.6910  Has0_F1_score: 0.6900  Non0_acc_2: 0.7027  Non0_F1_score: 0.7028  Mult_acc_5: 0.2930  Mult_acc_7: 0.2741  MAE: 1.1880  Corr: 0.4532 
> TEST (highest val acc) : >>  Has0_acc_2: 0.8586  Has0_F1_score: 0.8576  Non0_acc_2: 0.8841  Non0_F1_score: 0.8838  Mult_acc_5: 0.5918  Mult_acc_7: 0.5160  MAE: 0.5941  Corr: 0.8642  Loss: 2.3815 
> 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 343/343 [00:54<00:00,  6.34it/s]
> TEST >> loss:  2.4808    M_loss:  0.6027   T_loss:  0.586   A_loss:  1.2922
> M: >>  Has0_acc_2: 0.8499  Has0_F1_score: 0.8472  Non0_acc_2: 0.8811  Non0_F1_score: 0.8797  Mult_acc_5: 0.5685  Mult_acc_7: 0.4840  MAE: 0.6027  Corr: 0.8602 
> T: >>  Has0_acc_2: 0.8557  Has0_F1_score: 0.8541  Non0_acc_2: 0.8811  Non0_F1_score: 0.8803  Mult_acc_5: 0.5773  Mult_acc_7: 0.5000  MAE: 0.5860  Corr: 0.8677 
> A: >>  Has0_acc_2: 0.6516  Has0_F1_score: 0.6500  Non0_acc_2: 0.6524  Non0_F1_score: 0.6521  Mult_acc_5: 0.2449  Mult_acc_7: 0.2376  MAE: 1.2922  Corr: 0.3512 
> TEST (lowest val loss) : >>  Has0_acc_2: 0.8499  Has0_F1_score: 0.8472  Non0_acc_2: 0.8811  Non0_F1_score: 0.8797  Mult_acc_5: 0.5685  Mult_acc_7: 0.4840  MAE: 0.6027  Corr: 0.8602  Loss: 2.4808 
> ```
Hello, I have not yet replicated this paper. I noticed that you adjusted the batch size to 2, while the original paper uses 8 as the default. I wonder if this is the reason for your lower accuracy.
